src/libpriv/rpmostree-passwd-util.c

```diff
@@ -391,8 +391,8 @@ rpmostree_check_passwd_groups (gboolean         passwd,
                                                        ignore_removed_ents,
                                                        error))
         return FALSE;
+      ignore_all_removed = rpmostree_str_ptrarray_contains (ignore_removed_ents, "*");
```

Comment:
Minor: for a minimal diff, we don't really have to move this anymore now that `rpmostree_str_ptrarray_contains()` handles `NULL` arrays, right?